index.html

```diff
@@ -311,7 +343,7 @@ <h2 id="connect">Let's Connect</h2>
                 <span>CSS</span>
               </li>
             </ul>
-          </li>
+          </ul>
```

Comment:
Great job on fixing the width issue for the categories section. 🎉 
- [ ] [  **OPTIONAL** ] APPLICABLE FOR DESKTOP VERSION: I believe that the categories section seems to be left-aligned more than required. Please can you fix the alignment issue by referring the Figma template 👍🏽  

Your design 👇🏽 
<img width="1440" alt="Screenshot 2022-12-10 at 1 34 18 AM" src="https://user-images.githubusercontent.com/24832846/206788447-ad167a77-996b-44c0-a849-df6b50472cd2.png">

Figma design 👇🏽 
<img width="1132" alt="Screenshot 2022-12-10 at 1 39 06 AM" src="https://user-images.githubusercontent.com/24832846/206787995-dff3af13-ab26-44bf-8db4-92f718326c64.png">